index.php

```diff
@@ -1,63 +1,64 @@
 <?php
 
-
 $extensions = [
     'js' => 'node',
     'php' => 'php',
-    'py' => 'Python',
-    'javac' => 'java',
-];
-$keys = [
-    'firstName',
-    'lastName',
-    'language',
-    'id',
+    'py' => 'python',
+    'java' => 'java',
 ];
-
-$log_directory = $_SERVER['DOCUMENT_ROOT'] . 'scripts';
-// var_dump($log_directory);
-// exit;
+$count = 0;
+$log_directory = $_SERVER['DOCUMENT_ROOT'] . '/scripts';
```

Comment:
The apppending "/" in Document_root is missing in some php versions when run in browser